loaders/android/bootstrap.java.in

```diff
@@ -158,7 +158,7 @@ public class @SYS_APPNAME@ extends Activity implements @ANDROID_JAVA_IMPLEMENTS@
     // start EVENT_IDLE 
     if(idle_tmScheduleRate > 0) idle_tm.scheduleAtFixedRate(idle_task, 0, idle_tmScheduleRate);
 
-    nativeInstanceInit();
+    nativeInstanceInit(ln_getFilesDir());
```

Comment:
Sorry, but where is `ln_getFilesDir()` declared?

Reply:
ah, the other half missing.  :-/

in [bootstrap.java.in](https://github.com/part-cw/lambdanative/blob/master/loaders/android/bootstrap.java.in#L90) line 90ff. I have

      public String ln_getFilesDir() {
          return getFilesDir().toString();
     }